tests/Fixer/Phpdoc/PhpdocNullableTypeDeclarationFixerTest.php

```diff
@@ -0,0 +1,294 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Phpdoc;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
```

Comment:
we shall eat our own dog food. that's one of ways we ensure quality of solutions we provide.

i would love to see this rule temporarily enabled in `.php-cs-fixer.dist.php` and we see the result. if it's working smooth/crashing, if outcome diff is good, etc.